.github/workflows/static-analysis.yml

```
@@ -60,11 +62,10 @@ jobs:
       - name: "Run Psalm"
         run: "vendor/bin/psalm --show-info=false --stats --output-format=github --threads=$(nproc) --report=psalm.sarif"
 
+      - name: "Upload sarif report"
```

```suggestion
      - name: "Upload SARIF report"
```

Nite: SARIF is an acronym.

---

Changed